src/components/CheckEditor/FormComponents/HttpCheckAuthentication.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { MutableRefObject, useCallback, useRef, useState } from 'react';
```

Comment:
```suggestion
import React, { useCallback, useRef, useState } from 'react';
```

Just in case you want to accept the suggestion below without checking out.